tests/Functional/FunctionalTest.php

```diff
@@ -72,8 +89,13 @@ private function runSymfonyConsole(): Process
             // Run bin/console
             'tests/Functional/App/bin/console',
         ]);
-        $dockerCommand->mustRun();
+        $dockerCommand->run();
 
         return $dockerCommand;
     }
+
+    private function assertCommandIsSuccessful(Process $command): void
+    {
+        $this->assertTrue($command->isSuccessful(), $command->getOutput() . PHP_EOL . $command->getErrorOutput());
```

Comment:
I did something similar in #24, But I also made sure that the error output was empty.